app/views/pages/index.html.erb

```diff
@@ -97,8 +97,8 @@
     <% @top_pilots.each do |pilot| %>
       <div class="col-xs-12 col-md-9 col-lg-3 pilot">
         <%= image_tag pilot.avatar, class: 'img-circle' %>
-        <p><%= pilot.first_name %></p>
-        <%= link_to 'View profile', operator_profile_path(pilot.operator_profile), class: 'btn btn-md btn-primary' %>
+        <p class="pilotname"><%= pilot.first_name %></p>
+        <%= link_to 'View profile', operator_profile_path(pilot.operator_profile), class: 'viewall white' %>
       </div>
```

Comment:
i suggest keep working with the bootstrap buttons and change bootstraps defaults if necessary, but don't create new button classes